pandas/io/formats/templates/html.tpl

```diff
@@ -50,17 +50,17 @@
 {%- endblock thead %} 
 {%- block tbody %} 
 <tbody> 
-    {%- block before_rows %}{%- endblock before_rows %}
-    {%- for r in body %} 
-    {%- block tr scoped %}
-    <tr> 
-        {%- for c in r %} 
-        {%- if c.is_visible != False %} 
-        <{{ c.type }} id="T_{{ uuid }}{{ c.id }}" class="{{ c.class }}" {{ c.attributes|join(" ") }}>{{ c.display_value }}</{{ c.type }}>
-        {%- endif %} 
-        {%- endfor %} 
-    </tr> 
-    {%- endblock tr %}
+    {% block before_rows %}{% endblock before_rows %}
+    {% for r in body %}
```

Comment:
Does removing the opening `-` change the whitespace? Or was it redundant?

Reply:
It was redundant.

Reply:
I believe it was redundant.

Reply:
I think so too.